### PR TITLE
Adding Note regarding the pressure capabilities of the sensor. Changing the use of busio. Including learning guide.

### PR DESCRIPTION
**Pull request by jposada202020**
## Changes
Adding Note regarding the pressure capabilities of the sensor. 
Changing the use of busio. 
Including learning guide.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -61,18 +61,17 @@ Usage Example
 
     import board
     import digitalio
-    import busio
     import time
     import adafruit_bme280
 
     # Create library object using our Bus I2C port
```
Comment (dhalbert) [inline README.rst]:
```suggestion
    # Create sensor object, using the board's default I2C bus.
```

File: README.rst [hunk 2]
```diff
@@ -61,18 +61,17 @@ Usage Example
...
-    i2c = busio.I2C(board.SCL, board.SDA)
+    i2c = board.I2C()   # uses board.SCL and board.SDA
     bme280 = adafruit_bme280.Adafruit_BME280_I2C(i2c)
     #or with other sensor address
     #bme280 = adafruit_bme280.Adafruit_BME280_I2C(i2c, address=0x76)
 
     # OR create library object using our Bus SPI port
```
Comment (dhalbert) [inline README.rst]:
```suggestion
    # OR create sensor object, using the board's default SPI bus.
```

File: examples/bme280_normal_mode.py [hunk 3]
```diff
@@ -7,17 +7,15 @@
 Refer to the BME280 datasheet to understand what these parameters do
 """
 import time
-
 import board
-import busio
 import adafruit_bme280
 
 # Create library object using our Bus I2C port
```
Comment (dhalbert) [inline examples/bme280_normal_mode.py]:
```suggestion
# Create sensor object, using the board's default I2C bus.
```

File: examples/bme280_normal_mode.py [hunk 4]
```diff
@@ -7,17 +7,15 @@
...
-i2c = busio.I2C(board.SCL, board.SDA)
+i2c = board.I2C()  # uses board.SCL and board.SDA
 bme280 = adafruit_bme280.Adafruit_BME280_I2C(i2c)
 
 # OR create library object using our Bus SPI port
```
Comment (dhalbert) [inline examples/bme280_normal_mode.py]:
```suggestion
# OR create sensor object, using the board's default SPI bus.
```

File: examples/bme280_simpletest.py [hunk 5]
```diff
@@ -2,17 +2,15 @@
 # SPDX-License-Identifier: MIT
 
 import time
-
 import board
-import busio
 import adafruit_bme280
 
 # Create library object using our Bus I2C port
```
Comment (dhalbert) [inline examples/bme280_simpletest.py]:
```suggestion
# Create sensor object, using the board's default I2C bus.
```

File: examples/bme280_simpletest.py [hunk 6]
```diff
@@ -2,17 +2,15 @@
...
-i2c = busio.I2C(board.SCL, board.SDA)
+i2c = board.I2C()  # uses board.SCL and board.SDA
 bme280 = adafruit_bme280.Adafruit_BME280_I2C(i2c)
 
 # OR create library object using our Bus SPI port
```
Comment (dhalbert) [inline examples/bme280_simpletest.py]:
```suggestion
# OR create sensor object, using the board's default SPI bus.
```

**Review by dhalbert — CHANGES_REQUESTED** [review]

**Review by dhalbert — APPROVED** [review]
Looks good! Thanks! I did some slightly copy-editing from older phrasing.

**Comment by jposada202020** [comment]
thanks @dhalbert , I think we have a template, I would use this template and spread across the other library code, and examples, and readme.rst, etc.

**Comment by dhalbert** [comment]
I actually think the `# Create sensor object, using the board's default I2C bus.` comments are a bit extraneous. I think the code kind of speaks for itself. Maybe `Create sensor object, communicating over the board's default I2C bus`, if you want a comment like that.

**Comment by jposada202020** [comment]
Will do, Thanks

I'll use this two lines in the examples if needed
#### Create sensor object, communicating over the board's default I2C bus
#### OR Create sensor object, communicating over the board's default SPI bus

**Comment by jposada202020** [comment]
Thanks

**Pull request merged**